server/services/draftsService.ts

```diff
@@ -0,0 +1,74 @@
+import { Request } from 'express'
+import uuid from 'uuid'
+
+export interface Draft<Data> {
```

Comment:
nit: i'd find this more readable with a conventional type param e.g. `T`

Reply:
👍